config/global.json

```diff
@@ -140,6 +140,7 @@
     "polygonzkevm": "0x0000000000000000000000000000000000000000",
     "rootstock": "0x0000000000000000000000000000000000000000",
     "scroll": "0x0000000000000000000000000000000000000000",
+    "sonic": "0x039e2fB66102314Ce7b64Ce5Ce3E5183bc94aD38",
```

Comment:
This native address does not match with the native address in networks.json
The WETH was used accidentally here I believe.

Also we should remove the native addresses from global.json, too, right?

Reply:
Fixed and yes I think we should remove this.